### ISSUE TITLE
individual dosing scheme

### ISSUE THREAD

**Issue by RKasp** [Open-Systems-Pharmacology/Forum#260]
Dear all,
I am working on a population data set in which almost every subject ID has an individual dose varying with PD response, thus cannot be normalized by BW. Is there a possibility to work around cloning the simulation for each dose and individual? It seems to greatly lengthen run times for model optimization. Alternatively, I am considering to simulate a low, medium and high dose, however, I would only be able to use a fraction of data for parameter identification.
Your help is greatly appreciated. Thank you.

Best regards,

Rudiger

**Comment by StephanSchaller** [comment]
Dear Rudiger,

I guess it depends. If you do not think there is a BW dependence in you PK, you could post-hoc normalize all the doses by BW and then group them by the resulting normalized doses.
If the resulting doses are spread out continuously, you will not get around creating a simulation for each dose (if I understood your setup correctly).

Hope this helps,
Stephan

**Comment by StephanSchaller** [comment]
additional remark: if you group by low / medium / high and the doses in each group are not identical (BW normalized) you will in principal use wrong data for fitting most of your doses.

**Comment by RKasp** [comment]
Hi Stephan,

thank you for your input. I will explore further but you understood correctly, the doses are spread out continuously.  It seems I may need to take the more cumbersome route. The simulation of a low, medium and high dose was meant to be more descriptive. I have not explored PKSim in R much yet, but would it be possible theoretically to come up with a script to account for this situation? 

Best regards,

Rudiger

**Comment by StephanSchaller** [comment]
Hi Rudiger,

i would say the PK-Sim/R combo is the ideal alternative setup to do a post-hoc analysis of a clinical trial, if you are willing to do some scripting/coding in R.

You would only need to set up your model for one representative individual in PK-Sim, all the rest (loading data, settting the dose and exchanging subject physiology) can be done in R in a convenient (scripted) way.

Best,
Stephan

**Comment by RKasp** [comment]
Hi Stephan,
I will spend the time on R and get back to you soon.
Thank you for your help.
Regards,
Rudiger

**Comment by StephanSchaller** [comment]
>  If you go to R, you won't have access to the parameter identification

R has a number of packages for parameter identification (i.e. solving inverse problems for linear and nonlinear dynamical systems). One of these is the "FME" package.

But of course, if the number of individuals is not too many, staying withing PKSim is certainly the easier way, depending on your skills in R.

**Comment by Yuri05** [comment]
Closing. @RKasp Please reopen if you have any further questions on this topic

**Comment by RKasp** [comment]
Dear Stephan, Michael and all,
I would have a few more questions to this topic. I actually do need to run a parameter identification on a data set of 35 different doses, each dose has about 5-10 DVs. Due to the model set up, I run these in MoBi generating a file of about 90 MB, which is very difficult/slow to work with. All doses are here computed as separate simulations.

Is there a smart way around it in MoBi, unless running a parameter identification in R, which I may check into next?

Many thanks for your feedback.

Best regards,

Rudiger

**Comment by StephanSchaller** [comment]
Short answer: No.

But then, if you have different individuals you might need to go to Matlab, as `PKSimCreateIndividual` is not yet available for R.

**Comment by msevestre** [comment]
@RKasp 
> which is very difficult/slow to work with

What takes time? PI or Mobi? If the Parameter identification takes time, going to Matlab/R will not necessarily bring much as most time is spent calculating. 

Do you need to use all doses for the PI? Maybe you could split your data set into a training set for the PI and a test set to verify the PI output?

**Comment by msevestre** [comment]
@RKasp

Can you explain why you want to optimize all 32 simulations at the same time? Do you have adults or kids?  I think if we knew what is the question you are trying to solve with your model, we might be able to give you an easier way forward

**Comment by RKasp** [comment]
Hi Michael,
thank you for the quick reply and sorry to reply late. Do identify paramters over the complete wide dose range would give me an idea which process may get impeded or saturated at high doses so the model starts to deviate. I will try to find a route through binning with a mean dose, or run simulations senarios to assess a hypothesis.
Best, Rudiger

**Comment by msevestre** [comment]
Closing. Please reopen if you need more help
